runtime/include/chpl-gpu-impl.h

```diff
@@ -87,6 +88,15 @@ GPU_REDUCE(DECL_ONE_REDUCE_IMPL, maxloc)
 
 #undef DECL_ONE_REDUCE_IMPL
 
+#define DECL_ONE_SORT_IMPL(chpl_kind, data_type) \
+void chpl_gpu_impl_sort_##chpl_kind##_##data_type(data_type* data_in, \
+                                                  data_type* data_out, \
+                                                  int n, void* stream);
+GPU_SORT_TYPES(DECL_ONE_SORT_IMPL, keys)
+// TODO: GPU_SORT(DECL_ONE_SORT_IMPL, keysDesc/ DoubleBuffer/Pairs etc)
```

Comment:
cruft? Otherwise, I am not following what this TODO is about.

Reply:
Had to google cruft haha.
This TODO is just to demonstrate that there are more functions under cub::DeviceRadixSort that can be added later. These include decending sorts, DoubleBufferSorts (which use less auxillary space), and SortPairs which can sort key value pairs, and also potentially be extended to sort any type.

Reply:
Can you clarify that in the comment? Alternatively, you can drop the comment and create a private issue, too.